### PR TITLE
feat: added page routes

### PR DESCRIPTION
**Pull request by Kblob1**
## Describe your changes

_Provide some context for your changes and add images wherever applicable_

### Checklist
[] Unit tests added where applicable
[] Does not introduce any breaking changes
[] Ran `Prettier`

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+    // <Home />
+    <div>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/categories" element={<Categories />} />
+          <Route path="/account" element={<Account />} />
+        </Routes>
+      </BrowserRouter>
```
Comment (kevinleey) [inline client/src/App.js:19]: LGTM

File: client/src/components/Navbar.js [hunk 2]
```diff
@@ -19,16 +34,16 @@ function Navbar() {
           <nav>
             <ul>
               <li className="nav-item">
-                <a href="/home">HOME</a>
+                <button onClick={changeHome}>HOME</button>
               </li>
```
Comment (kevinleey) [inline client/src/components/Navbar.js:38]: Is there a reason why you didn't use the `Link` component that comes with `react-router-dom` but a button instead?

For example `<Link to="/example">EXAMPLE</Link>`

According to docs, I think this way you wouldn't need to use `nav()` and have those additional functions and the page should route automatically (For the sake of clarity the `Link` goes into an `li`)

File: client/src/pages/About.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import Navbar from "../components/Navbar";
+
+function About() {
+  return (
+    <div>
+      <Navbar />
+      <h1>About Section</h1>
+    </div>
+  );
+}
```
Comment (kevinleey) [inline client/src/pages/About.js:11]: Great start with these files

**Review by kevinleey — COMMENTED** [review]
Not too familiar with `react-router-dom` but had these comments as I was looking through documentation

File: client/src/components/Navbar.js (hunk 2 above)
Comment (Kblob1) [reply client/src/components/Navbar.js:38]: I did the buttons for simple css design. I can make it links and do a little touch up on it.

Comment (kevinleey) [reply client/src/components/Navbar.js:38]: Great, let me know if there are any blockers 👍

**Review by kevinleey — APPROVED** [review]
This is much better, thanks! As long as you can verify that the changes don't break the app and the navigation is working it LGTM :shipit: 

Also don't forget to add a short description to the PR!

**Review by jiaid — APPROVED** [review]
Looks good!

**Pull request merged**